scripts/merkl_incentive_campaign.py

```diff
@@ -0,0 +1,325 @@
+from datetime import datetime, timezone
+from eth_abi import encode_abi
+from brownie import interface, web3
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from helpers.constants import AddressZero, EmptyBytes32
+from rich.console import Console
+
+C = Console()
+
+"""
+The following methods are meant to provide the interface for the `DistributionCreator` contract
+
+## `distribute_incentives_concentrated_liquidity_pool` method:
+    - it is meant to be used to create a new campaign for incentivising a univ3 pool (less flexible method)
+    - incentive a selected univ3 pool
+    - set token to incentivise with
+    - determine the weights for each of the components (token A, token B & fees)
+    - timestamp when the campaign will kick off by passing date in string format UTC based
+    - length of the campaign defined in weeks
+
+## `create_campaign` method:
+    - more flexible as it allows us to put almost any configuration we want in the campaign data
+    - incentivize holders of an specific behaviour: erc20, CR of $eBTC holders
+
+Errors map:
+- InvalidReward(): typed error: 0x28829e82
+- NotSigned(): typed error: 0xa72952d8
+- CampaignSouldStartInFuture(): typed error: 0xcb737dcc
+- CampaignDurationBelowHour(): typed error: 0x1b5982fd
+- CampaignRewardTokenNotWhitelisted(): typed error: 0xc0460cfb
+"""
+
+HOURS_PER_DAY = 24
+SECONDS_PER_HOUR = 3600
+WEIGHTS_TOTAL_BASE = 10_000
+CAMPAIGN_TYPE_ERC20 = 1
+
+safe = GreatApeSafe(r.badger_wallets.ibbtc_multisig_incentives)
+
+# contracts
+distribution_creator = interface.IDistributorCreator(
+    r.merkl.distribution_creator, owner=safe.account
+)
+
+
+def distribute_incentives_concentrated_liquidity_pool(
+    incentive_token=r.assets.weth,
```

Comment:
FYI, there's a high chance that incentives are handled in the form of BADGER token as per treasury discussion. Any wETH would be swapped as needed for BADGER. It is ok to leave script as is since these swaps wouldn't happen atomically but rather through a Cow order as a prereq.

LGTM